react-native/setup/setup_firebase_multiple_envs.mo.md

```diff
@@ -0,0 +1,66 @@
+# [MO] Setup Firebase on iOS/Android with multiple environments *(~1h)*
+
+## Owner: Louis Lagrange
+
+## Prerequisites *(~30min)*
+
+- [ ] An Android or iOS app with the Firebase SDK installed
+   - For React Native apps, you can use [`react-native-firebase`](https://github.com/invertase/react-native-firebase)
+
+## Steps *(~30min)*
+
+It's pretty straightforward to install Firebase when you have only one environment.
+But what if you have multiple environments, such as `dev`, `staging` and `production`?
+Here's the steps by steps.
+
+### Android *(~10min)*
+On Android, Firebase groups the different configurations in one file.
+
+1. Go to the [Firebase console](https://console.firebase.google.com/)
+2. Select your project
+3. Click "Add an application", choose Android
+4. Fill in the form (giving a name like `Staging`), click save, and close the window
+5. Repeat steps 3 to 4 for all your environments
+5. Download the `google-services.json` file
+   1. Select one of the Android applications
+   2. Click on "google-services.json"
+6. Put `google-services.json` in your `app` folder (`android/app` in React-Native)
+7. You might want to add `google-services.json` in your `.gitignore`
+
+![steps](assets/firebase_android.png)
+
+### iOS *(~20min)*
```

Comment:
Would it be possible to split the big step in smaller steps (ideally 5 min) where you can check the intermediate result.

My goal is to form the bamer to be able to split there task in small ones instead of flying blind for a long time. 

The Django or the GraphQL standards are good exemple of those.